source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOBaseActionElement.mm

```diff
@@ -54,7 +54,9 @@ - (NSData *)additionalProperty
         std::stringstream sstream;
         writer->write(blob, &sstream);
         NSString *jsonString =
-        [[NSString alloc] initWithCString:sstream.str().c_str() encoding:NSUTF8StringEncoding];
+            [[NSString alloc] initWithCString:sstream.str().c_str()
+                                     encoding:NSUTF8StringEncoding];
+        delete writer;
```

Comment:
>delete writer; [](start = 8, length = 14)

All the other changes delete the writer right after the write call. Can we do that here as well for consistency?